docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -65,12 +75,12 @@ GET /_remote/info
 The API response indicates that the local cluster is connected to the remote
 cluster with the cluster alias `cluster_one`:
 
-[source,console-result]
+[source,console-result,subs=attributes+]
 ----
 {
   "cluster_one" : {
     "seeds" : [
-      "127.0.0.1:9300"
+      "127.0.0.1:{remote-interface-default-port}"
     ],
     "connected" : true,
     "num_nodes_connected" : 1,  <1>
```

Comment:
For RCS 2.0 remote cluster, this response contains an extra field `"cluster_credentials": "::es_redacted::"` which is important to determine whether a remote cluster is RCS 2.0 or 1.0. We need to somehow add it here.